pkg/infrastructure/services/deployment.go

```diff
@@ -20,11 +20,14 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strconv"
 )
 
 const (
 	portName      = "http"
 	singleReplica = int32(1)
+	// HTTPPortEnvKey Env variable to define port on which service will listen internally
+	HTTPPortEnvKey = "HTTP_PORT"
```

Comment:
Could you please reconsider exporting this constant?

Reply:
Currently this const is refered from `deployment_config.go` file. We can change its scope once we remove kogito app